packages/lib/models/Setting.ts

```diff
@@ -602,7 +602,7 @@ class Setting extends BaseModel {
 				},
 			},
 
-			startMinimized: { value: false, type: SettingItemType.Bool, section: 'application', public: true, appTypes: ['desktop'], label: () => _('Start application minimised in the tray icon') },
+			startMinimized: { value: false, type: SettingItemType.Bool, section: 'application', public: true, appTypes: ['desktop'], label: () => _('Start application minimised in the tray icon testing...') },
```

Comment:
Looks like you didn't mean to change this...

Reply:
I'm so sorry. It was meant to be for my testing and I pushed it accidently. I am fixing it asap

Reply:
I have pushed the changes again. Please review it.